go.mod

```diff
@@ -2,10 +2,11 @@ module github.com/netfoundry/ziti-sdk-golang
 
 go 1.14
 
-//replace github.com/netfoundry/ziti-foundation => ../ziti-foundation
+// replace github.com/netfoundry/ziti-foundation => ../ziti-foundation
 
 require (
 	github.com/Jeffail/gabs v1.4.0
+	github.com/cenkalti/backoff/v4 v4.0.2
```

Comment:
I didn't find where this was used

Reply:
I put comments on the two places it is used

Reply:
thanks, ziti.go was collapsed in the view so my search returned nothing